lib/core/src/lib/snackbar-content/snack-bar-data.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface SnackBarData {
+    actionLabel?: string;
```

Comment:
I think we should make sure that actionLabel is required when actionIcon is passed to mitigate potential future accessibility problems. What do you think?

Reply:
Sometimes actionLabel is not the same what actionIcon. For example if you remove file and you have notification with Undo action. Then you have Undo label which revert removing file and X icon which just closes notification if there is no need to revert removing. So there is no "relation" between both fields. Also sometimes you may not need label for example when you want just closing button. What do you think? Also we can ask @DenysVuika about opinion because I'm not sure too what approach is better :).

Reply:
Could you please explain it a bit more what you exactly mean because I'm not sure if I understand it correctly :) ?
Actual html looks like: 
`<p class="snackbar-message">{{ data.message }}</p>
<div *ngIf="data.showAction" class="action">
    <button mat-button (click)="snackBarRef.dismissWithAction()" *ngIf="data.actionLabel" class="action-label">
        {{data.actionLabel}}
    </button>
    <button mat-button *ngIf="data.actionIcon" (click)="onIconClicked()" class="action-icon">
        <mat-icon>{{ data.actionIcon }}</mat-icon>
    </button>
</div>`

So do you mean to add extra label to second button for icon? Like 
`{{somePassedLabelForIcon}}<mat-icon>{{ data.actionIcon }}</mat-icon>`

Then what if somebody will pass also label for first button? Won't we have too much content in action part? 

Or maybe do you mean to add label visible only for screen readers like aria-label? Please explain deeper what do you mean :)

Reply:
Oh sorry I meant the aria-label for screen readers :)

Reply:
Ah got it. Sure I think we can add aria-label for button with icon. Will add it in next commit in that PR.

Reply:
done